ecmd-core/pyapi/ecmdClientPyapi.i

```diff
@@ -76,6 +75,7 @@
 /*********** Start Files to swigify ***********/
 %include "ecmdDefines.H"
 %include "ecmdStructs.H"
+%apply std::string &OUTPUT { std::string & };    // Apply after ecmdStructs.H because this apply is too broad for ecmdStructs and will transform input parameters
```

Comment:
I see this modifying input and input/output string reference variables in functions like the following are being interpreted by this:

```
ecmdQueryScomGroup(..., std::string & io_scomGroupFileVersion)
getScomGroup(..., std::string & io_scomGroupFileVersion)
ecmdQueryFileLocation(..., std::string & io_version)
ecmdQueryBlock(..., const std::string & i_metadataHWPName = "all")
ecmdGetBlock(..., const std::string & i_metadataBlockEntry, ...)
makeSPSystemCall(..., const std::string & i_command, ...)
...
```